cli/cmd/sync_test.go

```diff
@@ -165,3 +329,21 @@ func TestSync_IsolatedPluginEnvironmentsInCloud(t *testing.T) {
 		})
 	}
 }
+
+func readSummaries(t *testing.T, filename string) []syncSummary {
+	p, err := os.ReadFile(filename)
+	assert.NoError(t, err)
+
+	lines := bytes.Split(p, []byte{'\n'})
```

Comment:
Does this test work OK on windows too?

Reply:
What is the behavior you would rather see?

Reply:
platform dependent line separators and a provision in this test to replace CRLF with LFs before proceeding :)